tests/addon/test_integration.py

```diff
@@ -7864,3 +7869,264 @@ def test_show_fsrs_optimization_reminder_skip_and_dont_show_again(
     # The config flag should now be False and saved
     assert config.public_config["remind_to_optimize_fsrs_parameters"] is False
     save_public_config_mock.assert_called_once()
+
+
+@skip_test_fsrs_unsupported
+@pytest.mark.qt_no_exception_capture
+def test_maybe_show_enable_fsrs_reminder(
+    anki_session_with_addon_data: AnkiSession,
+    qtbot: QtBot,
+    install_ah_deck: InstallAHDeck,
+    set_feature_flag_state: SetFeatureFlagState,
+    mocker: MockerFixture,
+    latest_instance_tracker: LatestInstanceTracker,
+):
```

Comment:
:pick: This test and the other added tests have arguments that are not used in the tests. We should remove the unused arguments
```suggestion
def test_maybe_show_enable_fsrs_reminder(
    anki_session_with_addon_data: AnkiSession,
    install_ah_deck: InstallAHDeck,
    set_feature_flag_state: SetFeatureFlagState,
    latest_instance_tracker: LatestInstanceTracker,
):
```